src/game-engine/world/actor/player/attack.ts

```diff
@@ -0,0 +1,19 @@
+//This will be used to pass information required to calulate attack and defense like weapon damage etc
+
+export class Attack {
+    damageType: AttackDamageType;
+    attackRoll: number =0;
+    defenseRoll: number = 0;
+    hitChance: number =0;
+    damage: number =0;
+    maximumHit: number;
+}
+
+export enum AttackDamageType {
+    Stab,
+    Slash,
+    Crush,
+    Magic,
+    Range,
+    None

```

Comment:
None is not an attack type, any reason why it's here? Just wondering.

Reply:
This was my null case- it just caused problems and i removed it from everywhere else - i guess i missed this one